ts/store/reducers/backendStatus/remoteConfig.ts

```diff
@@ -145,6 +145,26 @@ export const fimsServiceConfiguration = createSelector(
     )
 );
 
+/**
+ * Checks if a service should share iOS cookies in the FIMS flow.
+ * Returns true if the serviceId is NOT in the iOSCookieDisabledServiceIds list,
+ * false if it is in the list.
+ * If no configuration is available, returns true by default (cookies are shared).
+ */
+export const fimsServiceIdInCookieDisabledListSelector = createSelector(
```

Comment:
This selector returns a boolean, so there is no need to cache it. Can we rewrite it without the createSelector? Like
```
... = (state: GlobalState, serviceId: ServiceId) => pipe(
  state, 
  remoteConfigSelector, 
  O.chainNullableK(...), 
  O.map(...), 
  O.getOrElse(...)
)
```

Reply:
Should we add automatic tests for it?